docs/agent-bridging/spec.md

```diff
@@ -10,7 +10,7 @@ Desktop Agent Bridging is an experimental feature added to FDC3 in 2.1, hence, i
 
 :::
 
-<!-- ## TODO list
+\<!-- ## TODO list
```

Comment:
This escape is causing the comment to show in the final render of the markdown

Reply:
Hi @bingenito ,
Thank you for reviewing, let me check all the comments and other possible areas of similar issue.

We will try to fix and update the PR at the earliest. (before Citi Hackathon end date)